### DISCUSSION TITLE
how to upgrade from 2.7 to 3.8?

### DISCUSSION THREAD

**Discussion by harismateen** [chatwoot/chatwoot#9487] [Self hosted]
Hello, how to upgrade from 2.7 to 3.8?
env: ubuntu

**Comment by sojan-official** [comment]
@harismateen you will need to upgrade in iterations of a couple of versions each.  We generally recommend users to update every couple of months to stay in sync with latest versions. 

Please refer to https://www.chatwoot.com/docs/self-hosted/deployment/upgrade/
